core/src/main/java/com/example/core/components/RefreshableList.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.core.components
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.colorResource
+import com.example.core.R
+import com.google.accompanist.swiperefresh.SwipeRefresh
+import com.google.accompanist.swiperefresh.SwipeRefreshIndicator
+import com.google.accompanist.swiperefresh.rememberSwipeRefreshState
+
+@Composable
+fun RefreshableList(
+    size: Int,
+    onRefresh: () -> Unit,
+    isLoading: Boolean,
+    item: @Composable (Int) -> Unit
+) {
+    val swipeRefreshState = rememberSwipeRefreshState(isRefreshing = isLoading)
```

Comment:
isRefreshing is not the same as isLoading, as isLoading can occur during cases like initialization or searching.